Remotion/ObjectBinding/Web/Res/Themes/NovaGray/Html/BocList.css

```
@@ -276,6 +290,11 @@ th.bocListTitleCell > a[href]:active
   border: solid 1px #569CE0;
   background-color: #C5E0FC;
 }
+th.bocListTitleCell .bocListSortingOrder img
+{
+  height: 8px;
+  width: 13px;
+}
```

Blank line
```suggestion

th.bocListTitleCell .bocListSortingOrder img
{
  height: 8px;
  width: 13px;
}
```

Fascinating, that the original did not work.

---

When editing a row, the sorting icon is no longer an anchor.